sipyco/pc_rpc.py

```diff
@@ -24,6 +24,8 @@
 from sipyco.asyncio_tools import AsyncioServer as _AsyncioServer
 from sipyco.packed_exceptions import *
 
+CONNECTION_CLOSED_ERR = ConnectionAbortedError("Connection closed by the server")
```

Comment:
Why this variable?

Reply:
Because there are two places where it needs to be raised and I didn't want to have to repeat the message.